packages/react-vanilla-components/src/components/HCaptcha.tsx

```diff
@@ -0,0 +1,61 @@
+// *******************************************************************************
+//  * Copyright 2024 Adobe
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the “License”);
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *     http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an “AS IS” BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+
+//  * The BEM markup is as per the AEM core form components guidelines.
+//  * LINK- https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/hcaptcha/v1/hcaptcha/recaptcha.html
+//  ******************************************************************************
+
+import React from 'react';
```

Comment:
@navneet-ag 
can we add the HCaptcha component to the ReCaptcha files (https://github.com/adobe/aem-forms-headless-components/blob/main/packages/react-vanilla-components/src/components/ReCaptcha.tsx)? The only differences are the BEM and library. Everything else remains the same. We can add conditions based on the ':type'.

Reply:
Sure we can do this too.